### PR TITLE
added Shujie Fan to authors

### LINKED ISSUE #151: update authors in output
The logger output should indicate that @VOD555 has been doing the majority of the work on MDPOW for the last few years. Any other authors should be added, roughly "Shujie Fan, Ian Kenney, ...., Bogdan I. Iorga, Oliver Beckstein".

### PR DESCRIPTION
**Pull request by orbeckst**
fix #151

### CODE REVIEW

**Comment by orbeckst** [comment]
@iorga are you happy with having @VOD555 appear as the first author, given that he's really been the one keeping MDPOW alive? I'll merge once you approve the PR.

**Comment by iorga** [comment]
@orbeckst I completely agree with this.

**Comment by orbeckst** [comment]
@iorga thanks, could you then just do a review and approve? Thanks.

**Review by iorga — APPROVED** [review]

**Pull request merged**
